elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -0,0 +1,115 @@
+
+ul.connectedSortable {
+  /* This style applies to the MCQ options table, on the left hand side*/
+  border: 2px solid #A9A9A9; /* High visibility borders for accessibility */
+  border-radius: 4px;
+  background-color: #efefff; /* Pale Blue */
+
+  width: 100%;
+
+  margin: 0 40px 0 10px;
+
+  list-style-type: none;
+  padding: 5px 0 20px 0;
+  float: left;
+}
+
+div.right {
```

Comment:
The risk of class name conflicts is very high with generic names like `right`. Please add a `pl-order-blocks-` prefix to all CSS classes, so they will look like `pl-order-blocks-right` instead of `right`.

Reply:
fixed, variable renamed to your suggested name.